lib/article_json/export/common/html/elements/shared/caption.rb

```diff
@@ -0,0 +1,31 @@
+module ArticleJSON
+  module Export
+    module Common
+      module HTML
+        module Elements
+          module Shared
+            module Caption
+              # Generate the caption node
+              # @param [String] tag_name
+              # @return [Nokogiri::XML::NodeSet]
+              def caption_node(tag_name)
+                create_element(tag_name) do |caption|
+                  @element.caption.each do |child_element|
+                    caption.add_child(text_class.new(child_element).export)
+                  end
+                end
+              end
+
+              private
+
+              # @return [ArticleJSON::Export::Common::HTML::Elements::Base]
+              def text_class
```

Comment:
The name is a bit too generic for my taste, given that it's from a module. A class that uses this module could have a method `text_class` referring to CSS classes. So maybe something like `caption_text_exporter` or `caption_text_exporter_class`?

Reply:
Good point, I'll change this right away :)

Reply:
Done :)